testing/tests/DevExpress.ui.widgets.editors/checkbox.tests.js

```diff
@@ -144,12 +144,11 @@ QUnit.module('Checkbox', function() {
         });
 
         QUnit.test('checkbox icon must not resize according to the "width" and "height" options', function(assert) {
-            const newSize = 50;
-
             const $element = $('#checkBox').dxCheckBox();
             const instance = $element.dxCheckBox('instance');
             const initWidth = $element.find(ICON_SELECTOR).width();
             const initHeight = $element.find(ICON_SELECTOR).height();
+            const newSize = 50;
```

Comment:
?

Reply:
not really necessary, just moved it below, because it's a little strange to declare it in the very top and add an empty line after it.

Reply:
jfyi: it was written this way to visually separate **real** constant from another data

Reply:
ponyal